app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/ChatbotViewModel.kt

```diff
@@ -74,7 +81,7 @@ class ChatbotViewModel @Inject constructor(
 
   private fun handleMessageProcessing(
     updateConversation: () -> List<ChatInteraction>,
-    onSuccess: (GameGenieResponse, List<String>) -> Unit,
+    onSuccess: suspend (GameGenieResponse, List<String>) -> Unit,
```

Comment:
No need to hcange for now but just so you know, ideally in terms of architecture, viewmodel would communicate with a repository. This repository is the one that would make the requests part and map the response objects to the domain ones.